metrics-operator/controllers/analysis/controller.go

```diff
@@ -105,15 +107,15 @@ func (a *AnalysisReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	}
 
 	maps.Copy(res, done)
-	a.evaluateObjectives(ctx, res, analysisDef, analysis)
+	a.evaluateObjectives(res, analysisDef, analysis)
 	if err := a.updateStatus(ctx, analysis); err != nil {
```

Comment:
since we now report the full analysis in Prometheus, should we make sure to trim the raw status in case it is too big for a k8s resource 🤔 ? Overall limit should be[this](https://etcd.io/docs/v3.3/dev-guide/limit/#request-size-limit)

Reply:
only the score of the analysis is reported as a gauge metric, so the raw status will not be included anyway - see the example below:

![Screenshot 2023-09-20 at 14 44 27](https://github.com/keptn/lifecycle-toolkit/assets/2143586/0272a16e-5eb6-408a-b4ac-705b112fc53c)

Reply:
edit - and of course the objectives as well, but here also only the value of the metric:

![Screenshot 2023-09-21 at 09 29 10](https://github.com/keptn/lifecycle-toolkit/assets/2143586/0a7e488b-0893-47f9-b74a-613cf1d940e9)

Reply:
than let's think about it when it actually happens 😄